### PR TITLE
Tests are not running

### LINKED ISSUE #12: Integrate Test Runner in CI using Github Actions
The project currently has a GitHub Actions workflow set up to build the app. To enhance the development and review process, let's automate the running of tests for every pull request before merging. This ensures that no regressions are introduced with new code changes.

### TODO

1. Modify the existing GitHub Actions workflow to include a step for running tests.
2. Ensure that if tests fail, the workflow should mark the build as failed and provide appropriate logs for debugging.
3. Optional: Report test coverage, if any tools are integrated for that.

### Hint

A step for running tests: 
```yml
    - name: Run Tests
      run: ./mvnw test
```

### PR DESCRIPTION
**Pull request by aabarmin**
Closes #12

### CODE REVIEW

File: monolithic/monolithic-hexagonal/monolithic-hexagonal-loader/pom.xml [hunk 1]
```diff
@@ -9,10 +9,13 @@
         <version>0.0.1-SNAPSHOT</version>
     </parent>
 
-    <groupId>com.epam.microservices.monolithic</groupId>
```
Comment (naXa777) [inline monolithic/monolithic-hexagonal/monolithic-hexagonal-loader/pom.xml:12]: why? what's the effect of `groupId` removal?

File: monolithic/pom.xml [hunk 2]
```diff
@@ -9,7 +9,6 @@
         <version>0.0.1-SNAPSHOT</version>
     </parent>
 
-    <groupId>com.epam.microservices</groupId>
```
Comment (naXa777) [inline monolithic/pom.xml:12]: the same question. why `monolithic` project is excluded from the group?

File: pom.xml [hunk 3]
```diff
@@ -11,6 +11,13 @@
 
 	<packaging>pom</packaging>
 
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.4</version>
```
Comment (naXa777) [inline pom.xml:17]:
```suggestion
		<version>${spring-boot.version}</version>
```

**Review by naXa777 — CHANGES_REQUESTED** [review]

**Comment by naXa777** [comment]
I confirm that the main concern is fixed by your PR. Thank you :)

image

File: monolithic/monolithic-hexagonal/monolithic-hexagonal-loader/pom.xml (hunk 1 above)
Comment (aabarmin) [reply monolithic/monolithic-hexagonal/monolithic-hexagonal-loader/pom.xml:12]: It is the same as in the parent `pom.xml` so the declaration here is redundant.

File: monolithic/pom.xml (hunk 2 above)
Comment (aabarmin) [reply monolithic/pom.xml:12]: Just because it is the same as in the parent `pom.xml` so that it is redundant.

File: pom.xml (hunk 3 above)
Comment (aabarmin) [reply pom.xml:17]: No, this will not work because the parent `pom.xml` is resolved first before evaluating properties, so, let's leave it as is.

**Review by naXa777 — APPROVED** [review]

**Comment by naXa777** [comment]
looks good

**Pull request merged**